packages/server/src/commands/groceries/add_to_grocery_list_command.py

```diff
@@ -0,0 +1,32 @@
+from queue import Queue
+
+from src.commands.command import Command
+from src.commands.groceries.groceries_client import GroceriesClient
+
+
+class AddToGroceryListCommand(Command):
+    grammar_root: str = 'add-to-grocery-list'
+    grammar: str = """
+    add-to-grocery-list ::= "add " [^\n]+ " to the grocery list."
+    """
+
+    transcription_examples: list[str] = [
+        'add an eggplant to the grocery list.',
+    ]
+
+    def __init__(self, groceries_client: GroceriesClient):
+        self.groceries_client = groceries_client
+
+    def run(
+        self, command_text: str, response_q: Queue[str], sender_addr: tuple[str, int]
+    ) -> bool:
+        if 'to the grocery list' not in command_text:
```

Comment:
Since getting the item requires the command to end with " to the grocery list.", I suggest making this check the end instead of the whole string.
```suggestion
        if not command_text.endswith(' to the grocery list.'):
```

Reply:
:+1: Done